lib/apiservers/engine/backends/archive_test.go

```diff
@@ -235,7 +235,7 @@ func TestFindArchiveReaders(t *testing.T) {
 					rebase:  "etc",
 					strip:   "etc",
 					exclude: []string{""},
-					include: "etc/",
+					include: "etc",
```

Comment:
The actual target has the trailing slash in this case @hickeng but we will expect that a target in this case will generate an include without the trailing `/` since we still want the directory for `docker cp` behavior. we will still exclude things ending with a `/` but for docker cp the filter spec include should not have a `/` since the behavior is the same `/` or no.